src/remotebuild/lib/cli.ts

```diff
@@ -39,6 +40,9 @@ class CliHelper {
                 // if version flag found, show version and exit
                 CliHelper.handleVersionFlag(args);
             })
+            .then(function(): Q.Promise<any> {
+                return telemetry.init("REMOTEBUILD", require("../package.json").version, undefined ,"RemotebuildTelemetrySettings.json");
```

Comment:
I'd personally use null here instead of undefined, and change the code to make null have the proper semantics.